index.js

```diff
@@ -19,14 +19,20 @@ const monitoredChannels = { // channels you want to monitor ALL messages from AL
   [myUsername]: true
 }
 const monitoredTerms = [ myUsername ] // array of words, regex, etc you want to be notified of in joined channels
-const { chat } = new TwitchJs({ log: { level: 'silent' } })
+const ignoredTerms = [] // opposite of the above: array of words, regex, etc you want to *not* be notified of even in monitored channels
+
+const { chat } = new TwitchJs({
+  log: { level: 'silent' },
+  token: '',
```

Comment:
I don't think you should need this, it lets you "log in" anonymously as a justinfan with no token provided

Reply:
In a commit past the version named in package dot json, they changed some validation thing that requires `token` to be a string: https://github.com/twitch-js/twitch-js/blame/next/src/Api/utils/validators.ts#L13

If I pin twitch-js to beta.33 it works, but plain `npm install` left me with .42 and I think the behavior changes in .36. At first I changed package.json to require .33 specifically but I figure this is more forward-looking.